cordova-js-src/platform.js

```diff
@@ -56,9 +57,9 @@ module.exports = {
             // OR cordova.require('cordova/platform').activationContext
             // activationContext:{type: actType, args: args};
             var activationHandler = function (e) {
-                var args = e.detail.arguments;
-                var actType = e.detail.type;
-                platform.activationContext = { type: actType, args: args };
+                // Making all the details available as activationContext
+                platform.activationContext = utils.clone(e.detail); 
+                platform.activationContext.args = e.detail.arguments;       /* for backwards compatibility */
```

Comment:
Yes that is just for the backwards compatibility since I assume plugins would already be using activationContext.args property. Would changing it to `platform.activationContext.args = platform.activationContext.arguments` make any difference?